internal/controller/operator/factory/build/container_test.go

```diff
@@ -41,220 +41,206 @@ func (t testBuildProbeCR) ProbeNeedLiveness() bool {
 }
 
 func Test_buildProbe(t *testing.T) {
```

Comment:
changes to the container_test.go file looks unrelated to this PR.

Reply:
method for volumes and mounts creation was added to container.go, added tests for it to container_test.go and rewrote other tests to f-tests